src/index.html

```diff
@@ -1,13 +1,313 @@
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <div class="header__bottom">
+            <h1 class="header__title">
+              Your senses.
+              <br>
+              Set free.
+            </h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="menu page__menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img
+              src="images/logo.png"
+              alt="B&O logo"
+              class="top-bar__logo"
+            >
+          </a>
+          <a href="#" class="icon icon--close"></a>
+        </div>
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#recommended">Recommended</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#categories">Categories</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#about">About Us</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#contacts">Contact Us</a>
+              </li>
+            </ul>
+          </nav>
+          <div class="menu__contacts">
+            <a href="tel:+123455555555" class="menu__phone">+1 234 5555-55-55</a>
+            <a href="tel:+123455555555" class="menu__phone">call to order</a>
+          </div>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="main__content">
+        <div class="container">
+          <section class="recommended" id="recommended">
+            <h2 class="section-title">Recommended</h2>
+
+            <div class="recommended__products">
+              <article class="product recommended__product">
+                <a href="#" class="product__card">
+                  <img
+                    src="images/products/Beosound A5.png"
+                    alt="Beosound A5. Wi-Fi and Bluetooth speaker"
+                    class="product__photo"
+                    >
+                  <h3 class="product__title">Beosound A5</h3>
+                  <p class="product__desc">Huge sound anywhere – in the home or outdoors. Beosound A5 is a Wi-Fi and Bluetooth speaker with long-lasting battery and wireless charging built in.</p>
+                  <p class="product__price">$ 1299.00</p>
+                </a>
+              </article>
+
+              <article class="product recommended__product">
+                <a href="#" class="product__card">
+                  <img
+                    src="images/products/Beoplay HX.png"
+                    alt="Beoplay HX. Wireless noise-cancelling headphones"
+                    class="product__photo"
+                    >
```

Comment:
Fix code style everywhere
```suggestion
                  <img
                    src="images/products/Beoplay HX.png"
                    alt="Beoplay HX. Wireless noise-cancelling headphones"
                    class="product__photo"
                  >
```